src/test/system_tests/ceph_s3_tests/test_ceph_s3_deploy.sh

```diff
@@ -38,7 +38,7 @@ commit_epoch=$(git show -s --format=%ci ${CEPH_TESTS_VERSION} | awk '{print $1}'
 commit_date=$(date -d ${commit_epoch} +%s)
 current_date=$(date +%s)
 
-max_days="180"
+max_days="360"
```

Comment:
I think it was fixed already - https://github.com/noobaa/noobaa-core/pull/8191
so it can be removed

Reply:
done